dockers /Dockerfile

```
@@ -0,0 +1,25 @@
+FROM amazonlinux:2023
+RUN yum update 
+WORKDIR APP
+RUN yum install wget -y
```

could do this in one block
```suggestion
RUN yum update && \
  yum -y install \
  wget \
  tar \
  gzip && \
  yum clean all
```